Clean up defconfig warnings from our config sanity checker.

CONFIG_POWER_AVS has been removed, submitted to linux-arm-kernel@.

CONFIG_BACKLIGHT_GENERIC has been removed, backport of 717c4c8.

CONFIG_USB_CONN_GPIO needs to be y not m as it is depended on by PHY_TEGRA_XUSB
which is y.

Upstream-Status: Submitted
Signed-off-by: Ross Burton <ross.burton@arm.com>
---
 arch/arm64/configs/defconfig | 65 +++++++++++++-----------------------
 1 file changed, 24 insertions(+), 41 deletions(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 6d84761d1285..16367559b143 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -514,7 +514,6 @@ CONFIG_GPIO_PCA953X_IRQ=y
 CONFIG_GPIO_BD9571MWV=m
 CONFIG_GPIO_MAX77620=y
 CONFIG_GPIO_SL28CPLD=m
-CONFIG_POWER_AVS=y
 CONFIG_QCOM_CPR=y
 CONFIG_ROCKCHIP_IODOMAIN=y
 CONFIG_POWER_RESET_MSM=y
@@ -683,7 +682,6 @@ CONFIG_DRM_PANFROST=m
 CONFIG_FB=y
 CONFIG_FB_MODE_HELPERS=y
 CONFIG_FB_EFI=y
-CONFIG_BACKLIGHT_GENERIC=m
 CONFIG_BACKLIGHT_PWM=m
 CONFIG_BACKLIGHT_LP855X=m
 CONFIG_LOGO=y
@@ -727,7 +725,7 @@ CONFIG_SND_SOC_WSA881X=m
 CONFIG_SND_SIMPLE_CARD=m
 CONFIG_SND_AUDIO_GRAPH_CARD=m
 CONFIG_I2C_HID=m
-CONFIG_USB_CONN_GPIO=m
+CONFIG_USB_CONN_GPIO=y
 CONFIG_USB=y
 CONFIG_USB_OTG=y
 CONFIG_USB_XHCI_HCD=y
